Revoke workspace deletion request feature (#7372)
segmentio/segment-docs: src/privacy/account-deletion.md

**To delete all data from one workspace:**

1. Sign in to the Segment app, select the workspace you'd like to delete, and click **Settings**.
2. On the General Settings page, click the **Delete Workspace** button. 
3. Follow the prompts on the pop-up to delete your workspace. 

> After a 5 day cool-off period, Segment will irrevocably delete your workspace. 

Author: ```suggestion
Segment will irrevocably delete your workspace 5 days after you initiate your deletion request. 
```

> If you want to revoke the workspace deletion request during the 5 day cool-off period, open the [Workspace Settings](https://app.segment.com/goto-my-workspace/settings/basic){:target="_blank"} page, select the **General Settings** tab and click **Revoke Workspace Deletion**. 

Author: ```suggestion
If you want to revoke the workspace deletion request during the 5 days after you initiated your request, open the [Workspace Settings](https://app.segment.com/goto-my-workspace/settings/basic){:target="_blank"} page, select the **General Settings** tab and click **Revoke Workspace Deletion**. 
```
